Fastlane/Configuration.rb

```diff
@@ -0,0 +1,43 @@
+require 'yaml'
+
+class Configuration
+
+  attr_reader :build_number
+  attr_reader :app_scheme, :app_output_directory, :app_display_name
+  attr_reader :scan_devices
```

Comment:
We don't need any of the readers above as the methods already exist and are publicly accessible. We should however have them for `@lane_name` & `@yaml` so we don't need to call instance variables directly from within the class (+ encapsulate within private to limit access outside), e.g:

```
...
def initialize(lane_name, yaml)
    @lane_name = lane_name
    @yaml = YAML.load_file(yaml, aliases: true)[lane]
  end
...
private

attr_reader :lane_name, :yaml

def blah
  lane_name
end
```

Reply:
lol, so I have all the ones I don't need and none of the ones I do need? Classic!